packages/connect-web/src/impl/iframe/index.ts

```diff
@@ -0,0 +1,326 @@
+import EventEmitter from 'events';
```

Comment:
maybe `src/impl/core-in-iframe`

Reply:
and maybe index files are not needed? just make it `core-in-iframe.ts` and `core-in-popup.ts`?

Reply:
True, there's not much need for other files, so we can get rid of the subdirectories. `core-in-iframe` is also more descriptive.